net/oui/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+revision            0
+
+version             0.1.8
+name                oui
+
+homepage            https://github.com/thatmattlove/oui
```

Comment:
Don't set `version`, `name`, `homepage` manually; let `github.setup` do it for you.